handlers/datasetediting/department.go

```diff
@@ -5,11 +5,14 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/ugent-library/biblio-backoffice/ctx"
 	"github.com/ugent-library/biblio-backoffice/handlers"
 	"github.com/ugent-library/biblio-backoffice/models"
 	"github.com/ugent-library/biblio-backoffice/render"
 	"github.com/ugent-library/biblio-backoffice/snapstore"
+	"github.com/ugent-library/biblio-backoffice/views/dataset"
```

Comment:
I would alias this as views, there's a possible conflict with a dataset local variable
```
views "github.com/ugent-library/biblio-backoffice/views/dataset"
```

Reply:
Done